src/components/HelpPopup.vue

```diff
@@ -27,16 +30,34 @@ const showModal = () => {
   openedOnce.value = true;
 };
 const closeModal = () => dialog.value?.close();
+
+const translatedInput = ref<string>('');
+
+watch(
+  () => props.input,
+  (newValue) => {
+    if (newValue === 'expeditions') {
+      translatedInput.value = t('translation.expeditions');
+    } else if (newValue === 'stats') {
+      translatedInput.value = t('translation.stats');
+    } else if (newValue === 'traits') {
+      translatedInput.value = t('translation.traits');
+    } else {
+      translatedInput.value = 'Unknown value';
+    }
+  },
+  { immediate: true }
+);
```

Comment:
This seems like it could be done easier with a `computed` property and an object lookup